### PR TITLE
Updated README.md to help with setup

### PR DESCRIPTION
**Pull request by amritsiidhu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ To get the frontend running you may need to update the `PYTHONPATH` to pick up t
 export PYTHONPATH="$PYTHONPATH:<location of google-cloud-sdk>/platform/google_appengine:<location of google-cloud-sdk>/platform/google_appengine/lib/:<location of google-cloud-sdk>/platform/google_appengine/lib/yaml/"
 ```
 
+## Install requirements
+
+run `pip install -r requirements.txt`
```
Comment (kentsanggds) [inline README.md]: Would be nice to capitalise run in line with rest of doc

**Comment by kentsanggds** [comment]
There should also probably be a mention of the virtualenv and installation of dependencies in the README

**Comment by kentsanggds** [comment]
Note that dependencies are installed like this - `https://github.com/NewAcropolis/frontend/blob/deadc8bb19d3a6ccb14087221b110d504602337f/Makefile#L8`

File: README.md (hunk 1 above)
Comment (AmritSidhu) [reply README.md]: Removed this completely as the requirements are install through a make command

Comment (AmritSidhu) [reply README.md]: Also now mentioned the virtualenv which was mentioned in the api

**Pull request closed without merging**